lib/ngx/errlog.md

```diff
@@ -253,6 +254,108 @@ end
 
 [Back to TOC](#table-of-contents)
 
+rawlog
```

Comment:
We use snake case function names consistently in this module. I think it's better to be `raw_error_log`?

Reply:
Or `raw_log()`.

Reply:
`raw_log()` makes perfect sense to me. I will update!